raindrops.rb

```diff
@@ -0,0 +1,32 @@
+module BookKeeping
+  VERSION = 3
+end
+class Raindrops
+  def self.convert(in_number)
+    out_strings = ""
+    factor_arr = []
+    factor_arr = factor(in_number)
+    conditions = { 3 => "Pling", 5 => "Plang", 7 => "Plong"}
+    conditions.keys.each do |key|
+      (0...(factor_arr.length)).each do |f|
+        if key == factor_arr[f]
+          out_strings += conditions[key]
+        end
```

Comment:
make it one linear using ruby shorthand way. like..

do..this... if (condition)